sanity-io/structured-content-2022
# Speakers page

# Speakers page

## Intent

Page for individual Speakers that have at least one incoming reference, e.g. [Carrie Hane][carrie].
Solves [this](https://app.shortcut.com/sanity-io/story/14187/speaker-details-front-end) Shortcut Story.


## Testing this PR

1. Open /speakers/carrie-hane either locally or in the [latest Preview Deploy][carrie]
2. Verify that the page renders all fields correctly and appears in accordance with [the corresponding Figma sketches](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=4476%3A15890) for all supported viewports

[carrie]: https://structured-content-2022-web-git-feature-sc-14187speaker-88f7a5.sanity.build/speakers/carrie-hane
